main.py

```diff
@@ -576,6 +576,14 @@ def main(cl_arguments):
                         model, ckpt_path, args.cuda, skip_task_models=[], strict=strict
                     )
                     evaluate_and_write(args, model, [task], splits_to_write)
+            else:
```

Comment:
@pruksmhc - If this gets merged before your PR, you'll have to make sure these changes don't get incorporated into yours.